WcaOnRails/app/views/registrations/psych_sheet_event.html.erb

```diff
@@ -5,7 +5,7 @@
     <%= cubing_icon @event.id %> <%= @event.name %> Psych Sheet
   </h2>
   <p>
-  Sorted by: <%= @psych_sheet.sort_by %>, sort by <a href="<%= competition_psych_sheet_event_path(@competition, @event.id) %>?sort_by=<%= @psych_sheet.sort_by_second %>"><%= @psych_sheet.sort_by_second %></a>
+    Sorted by: <%= @psych_sheet.sort_by %><% if !@event.multiple_blindfolded? %>, sort by <%= link_to @psych_sheet.sort_by_second, competition_psych_sheet_event_path(@competition, @event.id, sort_by: @psych_sheet.sort_by_second) %><% end %>
```

Comment:
`if !abc` in Rails can be written as `unless abc`

Reply:
Chances are that our linter will even complain about this during CI checks :innocent: